website/docs/d/custom_dns_configuration_cluster_aws.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: custom_dns_configuration_cluster_aws"
+sidebar_current: "docs-mongodbatlas-datasource-custom_dns_configuration_cluster_aws"
+description: |-
+    Describes a Custom DNS Configuration for Atlas Clusters on AWS.
+---
+
+# mongodbatlas_custom_dns_configuration_cluster_aws
+
+`mongodbatlas_custom_dns_configuration_cluster_aws` describes a Custom DNS Configuration for Atlas Clusters on AWS.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find **group_id** in the official documentation.
+
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_custom_dns_configuration_cluster_aws" "test" {
+	project_id                  = "<project-id>"
+	enabled                     = true
+}
+
+data "mongodbatlas_custom_dns_configuration_cluster_aws" "test" {
+    project_id = mongodbatlas_custom_dns_configuration_cluster_aws.test.id
+}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) Unique identifier for the project.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
```

Comment:
Shouldn't this list an id as well like other data sources we have, i.e. we have the internal terraform id or is the id, as we have a ticket on, set to the project id?

Reply:
So it does export the project id?  I thought it was exporting an id like this one from access list "id - The Terraform's unique identifier used internally for state management."

Reply:
If it does yes, but this time it only exports the project id because there's other important data that will require request with other parameter like other resources or for import.